### ISSUE TITLE
Adding / listing a todo due next week / month does not work

### ISSUE THREAD

**Issue by peperunas** [ultralist/ultralist#126]
Hello!

I found this bug while playing around with `todolist`. Great tool!

Here is an example of what happens.

```
$ t add something due next week
Todo 1 added.

$ t l due next week 

 all	
 1	[ ]	 	something	
```

Cheers!

**Comment by gammons** [comment]
I think that should work.  It should choose 7 days from today..

**Comment by gammons** [comment]
I believe this was fixed with #208
